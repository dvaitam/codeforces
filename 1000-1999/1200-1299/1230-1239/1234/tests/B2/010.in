22 10
11 14 2 10 5 7 2 10 3 3 10 10 6 14 19 9 5 1 18 2 19 7
