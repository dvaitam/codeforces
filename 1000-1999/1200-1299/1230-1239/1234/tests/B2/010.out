10
18 1 9 19 6 3 7 5 10 2
